### ISSUE TITLE
Check for LDAP php module during prereq check

### ISSUE THREAD

**Issue by progman32** [partkeepr/PartKeepr#778]
Using php5.6 on Ubuntu 14.04.5 LTS. Prereq check was clean save warnings for apcu and max execution time. Upon the DB connectivity check, I got a Generic Error which had the following details:
```
0x7002: LDAP extension not loaded
```

I fixed it like this:
```
$ sudo apt-get install php5.6-ldap
$ sudo apachectl restart
```

Seems the availability of this package should be added to the prereqs check.

**Comment by yschroeder** [comment]
I had the same issue (error message) on Ubuntu 16.04.2 LTS with php7.0. I installed `php7.0-ldap` and it worked.

**Comment by naturalgeek** [comment]
Same issue here, using debian 8, and php5.6


<img width="971" alt="screen shot 2017-03-08 at 13 49 53" src="https://cloud.githubusercontent.com/assets/3707331/23704634/2ac1dbe0-0406-11e7-88fb-e8b5989a720f.png">

**Comment by westerterp** [comment]
Same issue here. Why is php-ldap a requisite?
Shouldn't I be able to set up PartKeepr without it if I am not planning on using it?

**Comment by felicitus** [comment]
@westerterp the reason is given here: https://github.com/partkeepr/PartKeepr/issues/590#issuecomment-181445083

**Comment by baradhili** [comment]
Closing this as solved in 1.4.0
